README.md

```diff
@@ -16,10 +16,11 @@ This action sets up a go environment for use in actions by:
 The V2 offers:
 - Adds GOBIN to the PATH
 - Proxy Support
-- stable input 
+- `stable` input 
+- Check latest version
 - Bug Fixes (including issues around version matching and semver)
 
-It will first check the local cache for a version match. If version is not found locally, It will pull it from `main` branch of [go-versions](https://github.com/actions/go-versions/blob/main/versions-manifest.json) repository and on miss or failure, it will fall back to the previous behavior of download directly from [go dist](https://storage.googleapis.com/golang).
+The action will first check the local cache for a version match. If a version is not found locally, it will pull it from the `main` branch of the [go-versions](https://github.com/actions/go-versions/blob/main/versions-manifest.json) repository and on miss or failure, will fall back to the previous behavior of downloading directly from [go dist](https://storage.googleapis.com/golang). To change the default behaviour please use the [check-latest input](#check-latest-version).
```

Comment:
```suggestion
The action will first check the local cache for a version match. If a version is not found locally, it will pull it from the `main` branch of the [go-versions](https://github.com/actions/go-versions/blob/main/versions-manifest.json) repository. On miss or failure, it will fall back to downloading directly from [go dist](https://storage.googleapis.com/golang). To change the default behavior, please use the [check-latest input](#check-latest-version).
```

Thanks for updating the README!  A small edit: I don't think it makes sense to refer to "the previous behavior" here because in time it won't be relevant anymore.  The release notes would be a better place to compare old and new behavior.

Reply:
Yes, sure. Thank you. Done.